.env.sample

```diff
@@ -7,19 +9,26 @@ DATABASE_URL_TEST=postgres://user:pass@localhost:5432/outline-test
 SECRET_KEY=F0E5AD933D7F6FD8F4DBB3E038C501C052DC0593C686D21ACB30AE205D2F634B
 PORT=3000
 REDIS_URL=redis://redis:6379
-SLACK_KEY=71315967491.XXXXXXXXXX
-SLACK_SECRET=d2dc414f9953226bad0a356c794XXXXX
 URL=http://localhost:3000
 DEPLOYMENT=hosted
 ENABLE_UPDATES=true
+
+# Third party credentials (optional)
+SLACK_KEY=71315967491.XXXXXXXXXX
```

Comment:
KEY and SECRET are actually mandatory right now for auth, but all others optional, right?